staging/src/k8s.io/apimachinery/pkg/util/sets/string.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
nit: This seems unrelated to this commit?

Reply:
I have to do that, cause CI runs failed if I used [generated boilerplate](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.generatego.txt) for no generated go file.

Reply:
But then it doesn't belong into the "[add benchmark for set](https://github.com/kubernetes/kubernetes/pull/112377/commits/c3b96072d881840d911f4af78ab05dfd63a4b651)" commit, does it?

Reply:
yes, I will clean up commits after every things going well.

Reply:
I would not change the copyright lines at all unless a lawyer told you to :)